sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -143,15 +156,20 @@ export interface CertificatePolicy {
     keyUsage?: KeyUsageType[];
     lifetimeActions?: LifetimeAction[];
     reuseKey?: boolean;
-    subject?: string;
-    subjectAlternativeNames?: SubjectAlternativeNames;
     readonly updatedOn?: Date;
     validityInMonths?: number;
 }
 
-// @public
-export module CertificatePolicy {
-    const Default: CertificatePolicy;
+// @public (undocumented)
+export interface CertificatePolicySubject extends CertificatePolicyAll {
+    subject?: string;
+    subjectAlternativeNames?: undefined;
```

Comment:
I'm trying to remove subjectAlternativeName from this one, and subject from CertificatePolicySubjectAlternativeNames, but it's complaining.

Reply:
It's like if I try to access either property at any time, it complains that it doesn't exist in the other. My approach so far to mitigate this is to make sure that the unwanted property is undefined at most. Please let me know if anyone knows a better solution.

Reply:
This is not relevant anymore! We can always have both properties.